### PR TITLE
Add default config.properties file for data2services drill

### PR DESCRIPTION
**Pull request by vemonet**
config.properties file is always the same when running for data2services, so we integrated it directly in the container. But the user can still provide his own config.properties file

### CODE REVIEW

**Comment by vemonet** [comment]
Now we don't generate a triple if the object value is empty. And a few modification to README instructions , and using a default config.properties start if no file provided to make it easier to bootstrap

File: Dockerfile [hunk 1]
```diff
@@ -9,6 +9,8 @@ WORKDIR $TMP_DIR
 
 COPY . .
 
+COPY config.properties /config.properties
```
Comment (amalic) [inline Dockerfile:12]: why into the root folder?

File: src/main/java/r2rml/model/TriplesMap.java [hunk 2]
```diff
@@ -186,8 +186,8 @@ public boolean generateTriples(
 						// maps (but not referencing object maps) to row
 						List<RDFNode> objects = new ArrayList<RDFNode>();
 						for(ObjectMap om : opm.getObjectMaps()) {
-							RDFNode o = om.generateRDFTerm(row); 
-							if(o != null)
+							RDFNode o = om.generateRDFTerm(row);
+							if(o != null && !o.toString().equals(""))
```
Comment (amalic) [inline src/main/java/r2rml/model/TriplesMap.java:190]: why this additional null check?

File: README.md [hunk 3]
```diff
@@ -8,10 +8,32 @@
...
+docker run -it --rm --link drill:drill -v /data/my_dataset:/data r2rml /config.properties
+```
+
+* Content of the `config.properties` file:
+
+```properties
+connectionURL = jdbc:drill:drillbit=drill:31010
+mappingFile = /data/mapping.ttl
+outputFile = /data/rdf_output.nq
+format = NQUADS
```
Comment (amalic) [inline README.md:27]: we don't need a default config file inside the container. what is the rationale behind this?

File: Dockerfile (hunk 1 above)
Comment (vemonet) [reply Dockerfile:12]: I wanted to have a example/default file for the data2services pipeline use case. But I agree that in the root folder it is not really clean. We should put it in /example/apachedrill_config.properties

File: README.md (hunk 3 above)
Comment (vemonet) [reply README.md:27]: Having an out of the box example for drill usage (not especially needed)

**Comment by amalic** [comment]
no need to merge

**Pull request closed without merging**